packages/server-wallet/src/message-service/socket-io-message-service.ts

```diff
@@ -22,13 +30,32 @@ export class SocketIOMessageService implements MessageServiceInterface {
     this.peers.push(socket);
   }
 
+  public setLatencyOptions(options: LatencyOptions): void {
+    this._latencyOptions = options;
+  }
+
   public async send(messages: Message[]): Promise<void> {
-    if (messages.length > 0) {
-      this.server.emit('message', messages);
+    if (this._frozen) {
+      this._messageQueue.push(...messages);
+    } else {
+      const shouldDrop = Math.random() > 1 - this._latencyOptions.dropRate;
```

Comment:
Interestingly enough `prettier` won't let me do that.
![image](https://user-images.githubusercontent.com/1620336/128559128-c5225cc3-1d96-4b78-a9c7-72a6826bb2d9.png)


Why do you think the statement would always be `false` ?

Reply:
> Are we planning on using the `SocketIOMessageService` in production? If so, how do we feel about test logic in a production service?

I was originally thinking about creating a separate `TestSocketIOMessageService` but I think it might make more sense to keep it simple and then create a production-ready message service when we need one.

Perhaps I should rename the existing `SocketIOMessageService` to `TestSocketIOMessageService` or some variation of that to indicate that it's not production-ready.